### ISSUE TITLE
edhec_risk_kit_206

### ISSUE THREAD

**Issue by linasfx** [RunarOesthaug/MA-crossover_strategies#2]
What is behind the `import edhec_risk_kit_206 as erk`
Basically `table = erk.summary_stats(df_return_of_strategy, ppp=252)` covers nearly all functions. If it is some personalized file it should be avoided.

**Comment by RunarOesthaug** [comment]
Yes it is a file containing functions to calculate the various measures in the table and return a summary table. I didn't write it so I chose not to publish it.
